scripts/tests/chiptest/__init__.py

```diff
@@ -39,67 +39,114 @@ class ManualTest:
         manualtests.add(item)
 
     return manualtests
 
 
+def _GetFlakyTests() -> Set[str]:
+    """List of flaky tests, ideally this list should become empty."""
+    return {
+        "Test_TC_OO_2_4.yaml"
+    }
+
+
+def _GetSlowTests() -> Set[str]:
+    """Generally tests using sleep() a bit too freely.
+
+       10s seems like a good threshold to consider something slow
+    """
+    return {
+        "DL_LockUnlock.yaml",                             # ~ 10 seconds
+        "TestSubscribe_AdministratorCommissioning.yaml",  # ~ 15 seconds
+        "Test_TC_CC_5_1.yaml",                            # ~ 30 seconds
+        "Test_TC_CC_5_2.yaml",                            # ~ 30 seconds
+        "Test_TC_CC_5_3.yaml",                            # ~ 25 seconds
+        "Test_TC_CC_6_1.yaml",                            # ~ 35 seconds
+        "Test_TC_CC_6_2.yaml",                            # ~ 60 seconds
+        "Test_TC_CC_6_3.yaml",                            # ~ 50 seconds
+        "Test_TC_CC_7_2.yaml",                            # ~ 65 seconds
+        "Test_TC_CC_7_3.yaml",                            # ~ 70 seconds
+        "Test_TC_CC_7_4.yaml",                            # ~ 25 seconds
+        "Test_TC_CC_8_1.yaml",                            # ~ 60 seconds
+        "Test_TC_DRLK_2_4.yaml",                          # ~ 60 seconds
+        "Test_TC_I_2_2.yaml",                             # ~ 15 seconds
+        "Test_TC_LVL_3_1.yaml",                           # ~ 35 seconds
+        "Test_TC_LVL_4_1.yaml",                           # ~ 55 seconds
+        "Test_TC_LVL_5_1.yaml",                           # ~ 35 seconds
+        "Test_TC_LVL_6_1.yaml",                           # ~ 10 seconds
+        "Test_TC_WNCV_3_1.yaml",                          # ~ 20 seconds
+        "Test_TC_WNCV_3_2.yaml",                          # ~ 20 seconds
+        "Test_TC_WNCV_3_3.yaml",                          # ~ 15 seconds
+        "Test_TC_WNCV_3_4.yaml",                          # ~ 10 seconds
+        "Test_TC_WNCV_3_5.yaml",                          # ~ 10 seconds
+        "Test_TC_WNCV_4_1.yaml",                          # ~ 20 seconds
+        "Test_TC_WNCV_4_2.yaml",                          # ~ 20 seconds
+        "Test_TC_WNCV_4_5.yaml",                          # ~ 12 seconds
+    }
+
+
+def _GetInDevelopmentTests() -> Set[str]:
+    """Tests that fail in YAML for some reason.
+
+       Goal is for this set to become empty.
+    """
+    return {
+        "Test_TC_ACL_2_10.yaml",
+        "Test_TC_ACL_2_7.yaml",
+        "Test_TC_ACL_2_8.yaml",
+        "Test_TC_ACL_2_9.yaml",
+        "TestEvents.yaml",
+        "Test_TC_ACE_1_1.yaml",
+        "Test_TC_ACE_1_5.yaml",
+        "Test_TC_SC_5_1.yaml",
+        "Test_TC_SC_5_2.yaml",
+        "TestCommissionerNodeId.yaml",
+        "TestClusterMultiFabric.yaml",
+        "TestGroupMessaging.yaml",
+        "TestMultiAdmin.yaml",
```

Comment:
This lost the documentation about what the reason is that each test is in the list.  Please add those back, as comments?

Reply:
Ah ... did not notice notes were added in the mean time. Updated.